Source/DafnyCore/AST/Modules/AliasModuleDecl.cs

```diff
@@ -17,6 +18,14 @@ public AliasModuleDecl(Cloner cloner, AliasModuleDecl original, ModuleDefinition
     : base(cloner, original, parent) {
     if (original.TargetQId != null) { // TODO is this null check necessary?
       TargetQId = new ModuleQualifiedId(cloner, original.TargetQId);
+
+      /*
+       * Refinement cloning happens in PreResolver, which is after the ModuleQualifiedId.Root fields are set,
+       * so this field must be copied as part of refinement cloning.
+       * However, letting refinement cloning set CloneResolvedFields==true causes exceptions for an uninvestigated reason,
+       * so we will clone this field even when !CloneResolvedFields.
```

Comment:
Sounds like just a performance hit to clone despite this flag, so this seems safe to me.